src/main/java/com/rarchives/ripme/utils/Utils.java

```diff
@@ -269,7 +269,11 @@ public static void clearURLHistory() {
      * Return the path of the url history file
      */
     public static String getURLHistoryFile() {
-        return getConfigDir() + File.separator + "url_history.txt";
+        if(getConfigString("history.location", "").length()==0) {
```

Comment:
This can be rewritten as 

```
return getConfigString("history.location", getConfigDir() + File.separator + "url_history.txt");
```

If the first arg is unset in the config file getConfigString will return the 2nd arg

Reply:
Derp, I should've thought about that. Thanks!